sidebars.json

```diff
@@ -35,7 +35,8 @@
             "concepts-broker-load-balancing-features",
             "concepts-broker-load-balancing-benefits",
             "concepts-broker-load-balancing-concepts",
-            "concepts-broker-load-balancing-types"
+            "concepts-broker-load-balancing-types",
+            "concepts-broker-load-balancing-migration"
```

Comment:
For reviewers: I know here is not the perfect place, but there is no better place to locate this tutorial, so I put all types of docs under the same subject "Broker Load Balancing". I'll adjust it once there is a better solution.

<img width="305" alt="image" src="https://github.com/apache/pulsar-site/assets/50226895/8a41222b-a43f-4408-85e7-b43654e3c7e6">